.editorconfig

```diff
@@ -0,0 +1,23 @@
+# EditorConfig is awesome:http://EditorConfig.org
```

Comment:
Did you talk to @pranavkm and @anurse to make sure these settings are the same as elsewhere?